### PR TITLE
Add QT Creator and Windows dump files to .gitignore

### PR DESCRIPTION
**Pull request by swagtoy**
Add compact, short information about your PR for easier understanding:

- Goal of the PR: Fix Minetest
- How does the PR work? Sends a cool email to sfan5
- Does it resolve any reported issue? Between 1947 and 1969, at the height of the Cold War, more than 12,000 UFO sightings have been reported in the US.
- Does this relate to a goal in [the roadmap](https://github.com/minetest/minetest/blob/master/doc/direction.md)? idc
- If not a bug fix, why is this PR needed? it helps me. What usecases does it solve? it helps me.

## To do

Fix the gaem

- [ ] Delete Emacs
- [ ] Delete Irrlicht
- [ ] Delete formspec
- [ ] Sfan5 adds mobs api to engine
- [ ] Fix the Game

## How to test

Meet me at 27°22’50.10″N, 33°37’54.62″E.

### CODE REVIEW

**Comment by appgurueu** [comment]
> Meet me at 27°22’50.10″N, 33°37’54.62″E.

Alright, I'll test this PR in a year then :saluting_face:

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,8 @@ tags
 !tags/
 gtags.files
 .idea
+# QT Creator
```
Comment (appgurueu) [inline .gitignore:29]: no way

**Review by appgurueu — APPROVED** [review]
jokes aside there's nothing wrong with this except the fact it's maybe trying a bit too hard to be humorous (appreciated btw)

**Comment by swagtoy** [comment]
The PR was so simple i didnt feel a need to write anything special. Sorry about that. I will refrain in the future.

**Comment by keepitupkitty** [comment]
use gcc

**Pull request merged**